.circleci/config.yml

```diff
@@ -73,6 +80,8 @@ jobs:
       - run:
           name:  "Test: Execute Frontend  OWASP ZAP Vulnerability Scan"
           command: "cd tdrs-frontend; ./zap-scanner.sh"
+      - store_artifacts:
```

Comment:
Stores the reports for 30 days in a Circle CI managed S3 instance, which can be viewed in the `Artifacts` tab of the Job page in the Circle CI UI:
https://circleci.com/docs/2.0/artifacts/#uploading-artifacts